containers/deploy/service-accounts/tasks/main.yml

```diff
@@ -2,26 +2,18 @@
 - name: Login as system admin
   command: oc login -u system:admin
   when: minishift or cluster_up
-  tags:
-    - start
 
 - name: Add anyuid scc to anyuid service account
```

Comment:
Probably worth it to separate these from the lower-permissioned changes done as the developer user, and document them as requirements since many/most users probably can't log in as system:admin